mapcompose/src/main/java/ovh/plrapps/mapcompose/ui/state/MapState.kt

```diff
@@ -29,7 +30,8 @@ class MapState(
     tileStreamProvider: TileStreamProvider,
     tileSize: Int = 256,
     workerCount: Int = Runtime.getRuntime().availableProcessors() - 1,
-    magnifyingFactor: Int = 0
+    magnifyingFactor: Int = 0,
+    mapBackground: Color = Color.White
```

Comment:
If done like that, the background cannot be changed dynamically.
You could do that using an API (an extension property on MapState, with a setter).

Reply:
Fixed